Source/SpeechToTextV1/SpeechToText.swift

```diff
@@ -202,11 +199,11 @@ public class SpeechToText {
             username: username,
             password: password,
             model: model,
-            learningOptOut: learningOptOut,
-            serviceURL: serviceURL,
-            tokenURL: tokenURL,
-            websocketsURL: websocketsURL
+            learningOptOut: learningOptOut
         )
```

Comment:
Minor style suggestion, but I think it would be nice to add a line of whitespace here to delineate the sections of code.